### ISSUE TITLE
Error when trying to upload PSD files

### ISSUE THREAD

**Issue by mrlss** [avocode/avocode#608]
IMPORTANT: Please try to describe how did it happen.

-----
App v.2.5.2
Source UID: 122962
```
Error: invalid stored block lengths
  at Zlib._handle.onerror (zlib.js:345:17)

```

**Comment by vojtatranta** [comment]
This is duplicated issue, more info to be found here #607

**Comment by petrbrzek** [comment]
This error should be now fixed with the latest Avocode 2.5.3. Please try it and let us know.
